packages/language-picker/src/language-picker.tsx

```diff
@@ -145,21 +139,46 @@ function LanguagePicker< TLanguage extends Language >( {
 
 	const searchPlaceholder = __( 'Search languages…' );
 
+	const openMobileSearch = ( event: KeyboardOrMouseEvent ) => {
+		event.preventDefault();
+
+		setIsMobileSearchOpen( true );
+		searchInput.current?.focus();
+	};
+	useEffect( () => {
+		if ( searchInput.current && isMobileSearchOpen ) {
+			searchInput.current.focus();
+		}
+		if ( ! isMobileSearchOpen ) {
+			searchInput.current?.blur();
+		}
+	}, [ isMobileSearchOpen ] );
+
+	const closeMobileSearch = ( event: KeyboardOrMouseEvent ) => {
+		if ( event.type === 'keydown' && 'key' in event && event.key !== 'Enter' ) {
+			return;
+		}
+		event.preventDefault();
+		// if clicked or pressed enter we reset the form
+		setSearch( '' );
+		setIsMobileSearchOpen( false );
+		handleSearchClose();
+	};
 	return (
-		<Flex className="language-picker-component" align="left" direction="column" expanded={ false }>
-			<FlexBlock className="language-picker-component__heading">
-				<Flex justify="space-between" align="left">
-					<FlexItem className="language-picker-component__title wp-brand-font">
+		<VStack alignment="left" className="language-picker-component">
+			<VStack className="language-picker-component__heading">
+				<VStack alignment="left" style={ { justifyContent: 'space-between' } }>
```

Comment:
The stack components have props for this, although it isn't super obvious

```suggestion
				<VStack alignment="left" justify="space-between">
```

Reply:
Thank you for pointing this out!